### PR TITLE
Fix compiler warning about unnecessary string concatenation and runtime warning about missing keys.

### PR DESCRIPTION
**Pull request by jeffslofish**

### CODE REVIEW

**Comment by chris-wolff** [comment]
@jeffslofish In the case of `fetch(BASE_URL + "games/" + "1"` I actually liked having the concat there because it is going to eventually be `+ somevariable` that allows client to target a specific TTT game on the server. However, I agree with the other useless cat you found. If you modify your PR or make another PR to take out the one I mentioned, I will merge.

File: tic-react/src/js/Tile.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ class Tile extends Component {
     
     sendMovetoTTTServer(){
       //currently send the move to the first game in the python games hash
-      fetch(BASE_URL + "games/" + "1",
```
Comment (chris-wolff) [inline tic-react/src/js/Tile.js]: This one here, I'd actually like to keep the concat. If you want to go above and beyond the call of duty and create a variable and temporarily set to 1, or suggest a better structure for targeting specific games in the future, I'm open to suggestions!

Comment (chris-wolff) [reply tic-react/src/js/Tile.js]: Doing a variable would also shut up the warning, I think.

**Comment by jeffslofish** [comment]
How about using url-join as recommended here: https://stackoverflow.com/questions/16301503/can-i-use-requirepath-join-to-safely-concatenate-urls.

**Comment by chris-wolff** [comment]
I think that will work for now!

**Pull request merged**